spanner/spansql/parser.go

```diff
@@ -1286,6 +1289,50 @@ func (p *parser) parseAlterTable() (*AlterTable, *parseError) {
 	}
 }
 
+func (p *parser) parseAlterDatabase() (*AlterDatabase, *parseError) {
+	debugf("parseAlterDatabase: %v", p)
+
+	/*
+			ALTER DATABASE database_id
```

Comment:
The indentation for this entire comment is a bit weird. Please take a look at `parseAlterTable`.

Reply:
Done.